app/views/unattended/provisioning_templates/snippet/preseed_kernel_options.erb

```diff
@@ -0,0 +1,55 @@
+<%#
+kind: snippet
+name: preseed_kernel_options
+model: ProvisioningTemplate
+snippet: true
+description: options for the kernel / preseed startup initialization
+-%>
```

Comment:
It may be good to point out what host parameters this snippet considers since that information was dropped from the toplevel template. Or it may be good to add it to all those which use this snippet.